acquisitions/src/main/resources/thunderjet/mod-orders/features/piece-deletion-restriction.feature

```diff
@@ -0,0 +1,115 @@
+Feature: Piece deletion restrictions from order and order line
+
+  Background:
+    * url baseUrl
+    * callonce loginAdmin testAdmin
+    * def okapitokenAdmin = okapitoken
+    * callonce loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json' }
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*' }
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+
+  Scenario: Create finances
```

Comment:
Maybe move this to each scenario as we usually do in our newest tests, also @Positive/@Negative can be added